.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: false
+
+contact_links:
+  - name: Get help/ask a question in the Pyomo forum
+    url: https://groups.google.com/g/pyomo-forum
+    about: Have a question? Need some help from the community? The quickest way to get help is on Pyomo's forum.
```

Comment:
I don't know how this is rendered, but should we also include an entry for StackOverflow?

Reply:
If you go look at my fork (https://github.com/mrmundt/pyomo/issues/new/choose), it has the words and a link to the external site. What would you like the language to be for StackOverflow?

Reply:
Maybe we should link to this section of our main README instead of the google forum: https://github.com/Pyomo/pyomo#getting-help 

This way we could have a single link to both resources and once centralized place describing how to get help.

(we might also want to modify this section of the README to only list StackOverflow and the Pyomo forum along with a brief description of the types of questions are best asked in each place)